src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.links.strings

```diff
@@ -30,6 +30,6 @@
 
 "HealthCertificate_Info_register_FAQLink" = "https://www.coronawarn.app/en/faq/";
 
-"HealthCertificate_Error_FAQ_Link" = "https://www.coronawarn.app/en/faq/#vac_cert_invalid";
+"HealthCertificate_Error_FAQ_Link" = "https://www.coronawarn.app/en/faq/#hc_prefix_invalid";
```

Comment:
The button says "FAQ zu Impfzertifikaten"/"FAQ for Vaccination Certificates" and the text above is "Weitere Informationen zum Erhalt Ihres Impfzertifikats finden Sie in den FAQ."/"For more information about how to receive your vaccination certificate, please see our FAQ page.".

Will these texts be updated as well? And what about the German link?

Reply:
Ah, I see that there is a separate PR for the German link